### ISSUE TITLE
Plot too small when using facet and `draw!(f[1, 1], layer)`

### ISSUE THREAD

**Issue by rikhuijzer** [JuliaPlots/AlgebraOfGraphics.jl#304]
```
n = repeat(["a", "b", "c", "d"], 250)
y = rand(1000)
nt = (; n, y)

layer = data(nt) * mapping(:y; row=:n)
f = Figure()
draw!(f[1, 1], layer)
current_figure()
```
gives

![image](https://user-images.githubusercontent.com/20724914/141087757-6656ef0b-84da-4396-bc42-a28892f35680.png)

Same holds for `col`. But, removing `row` in `mapping` does work. It gives

![image](https://user-images.githubusercontent.com/20724914/141087834-e0785198-b8f7-43bc-9527-ea4a45699b1f.png)

EDIT: This is with AlgebraOfGraphics 0.6.0 and CairoMakie 0.6.6. I used to have this exact code running with AoG 0.5 without problems.

**Comment by piever** [comment]
I'm actually having some trouble reproducing this. Could you check if this is fixed in AlgebraOfGraphics 0.6.2 (latest version)?

**Comment by rikhuijzer** [comment]
> Could you check if this is fixed in AlgebraOfGraphics 0.6.2 (latest version)?

It is :tada:

## Details

I got scared for a second there that I created a faulty MWE, but luckily that turned out to not be the case.

In Pluto, doing:

```julia
begin
	using Pkg: Pkg
	Pkg.activate(; temp=true)
	packages = [
		Pkg.PackageSpec(; name="AlgebraOfGraphics", version="0.6.0"),
		Pkg.PackageSpec(; name="CairoMakie", version="0.6.6")
	]
	Pkg.add(packages)
end
```
```julia
using AlgebraOfGraphics, CairoMakie
```
```julia
begin
    n = repeat(["a", "b", "c", "d"], 250)
    y = rand(1000)
    nt = (; n, y)
    
    layer = data(nt) * mapping(:y; row=:n)
    f = Figure()
    draw!(f[1, 1], layer)
    current_figure()
end
```

gave the same wrong output.

Like stated above. The problem is fixed in the newest version of AlgebraOfGraphics.
